BUILDING.md

```diff
@@ -28,7 +28,7 @@ On FreeBSD and OpenBSD, you may also need:
 * libexecinfo (FreeBSD and OpenBSD only)
 
 
-```text
+```sh
```

Comment:
Yep these are all [aliases](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L3495-L3503). I'd prefer we keep it `sh` for simplicity.